### ISSUE TITLE
Is there any update for stable version (Device GPU is not working)

### ISSUE THREAD

**Issue by Lynchez** [apple/tensorflow_macos#124]
This is really annoying . Why cpu is so much faster than a gpu ? .I cannot even train the resnet50 model in cifar10 dataset when I use gpu. I hope we get a stable version soon because I think m1 is overrated here.

**Comment by ysimonx** [comment]
I run https://www.tensorflow.org/tutorials/quickstart/beginner

 360us by item on my macbook air m1
 700us by item on my pc windows 10 ryzen 6 3600 + NVIDIA GeForce 1660
 3000us by item on my macbook pro intel 7

i don't think m1 is overrated 

Instead of using GPU, it uses the 16-core neural engine

_"Recently Apple released the new M1 "system on a chip," which not only contains a built-in GPU, but also includes a 16-core "Neural Engine" which supports 11 trillion operations per second. Apple claims the Neural Engine will support up to 15x improvement in ML computation."_
source [https://www.infoq.com/news/2020/11/apple-tensorflow-acceleration/](https://www.infoq.com/news/2020/11/apple-tensorflow-acceleration/)

**Comment by ysimonx** [comment]
can you tell me which jupyter notebook you don't manage to run ? i wonder which issue you have

**Comment by Lynchez** [comment]
Hi, 
`from tensorflow.python.compiler.mlcompute import mlcompute
import tensorflow as tf
# Select GPU device.
mlcompute.set_mlc_device(device_name='gpu') # Available options are 'cpu', 'gpu', and ‘any'.
from tensorflow.python.framework.ops import disable_eager_execution
disable_eager_execution()
print(tf.executing_eagerly())`
I choose gpu with this code and keras.datasets.cifar10.load_data() and tf.keras.applications.resnet.ResNet50 model and There is no classic difference in the rest. Maybe resnet50 is not compatible for the current version

**Comment by Lynchez** [comment]
I try https://www.tensorflow.org/tutorials/quickstart/beginner
when ı choose cpu 
Epoch 1/5
60000/60000 [==============================] - 1s 16us/sample - loss: 0.3310 - accuracy: 0.9023

but i on same task with gpu
Epoch 1/5
60000/60000 [==============================] - 9s 145us/sample - loss: 0.2954 - accuracy: 0.9150

cpu is 10 times faster than gpu selection i think this tensorflow_macos has a real problems.

**Comment by ysimonx** [comment]
@Lynchez 

you must not have the same config as me

I run the same script with my apple macbook air m1, 16 Gb

when i try with "gpu", i am told that my script is switched to "cpu"

<img width="696" alt="Capture d’écran 2021-01-09 à 17 58 56" src="https://user-images.githubusercontent.com/1449867/104103795-5a6ae080-52a4-11eb-8cde-18440dd9275a.png">

**Comment by Lynchez** [comment]
Interesting , Mine macbook pro 8gb vers. 
I installled python3.8.6 with miniforge(ARM64)version.
I hope it will be updated soon

**Comment by ysimonx** [comment]
@Lynchez 

i use the python version provided with mac osx big sur
